etc/template/java-11-openjdk-amd64.yaml

```diff
@@ -0,0 +1,20 @@
+# glibc backend openjdk
+targets:
+  # copy jvm
+  - target: /usr/lib/jvm
+    copy:
+      - dirs:
+        - /usr/lib/jvm/java-11-openjdk-amd64
+  # emplicit copy java to enable autodep for java
+  - target: /usr/lib/jvm/java-11-openjdk-amd64/bin
+    copy:
+      - files:
+        - /usr/lib/jvm/java-11-openjdk-amd64/bin/java
+  - target: /usr/lib/jvm/java-11-openjdk-amd64/lib/server
+    copy:
+      - files:
+        - /usr/lib/jvm/java-11-openjdk-amd64/lib/server/libjvm.so
+  - target: /etc
```

Comment:
Can you please move this target to line 8, before the copy files?

Reply:
fixed.